src/Umbraco.Web/HealthCheck/NotificationMethods/EmailNotificationMethod.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Net.Mail;
+using System.Threading.Tasks;
+using Umbraco.Core.Configuration.HealthChecks;
+
+namespace Umbraco.Web.HealthCheck.NotificationMethods
+{
+    [HealthCheckNotificationMethod("email")]
+    public class EmailNotificationMethod : NotificationMethodBase, IHealthCheckNotificatationMethod
+    {
+        public EmailNotificationMethod(bool enabled, bool failureOnly, HealthCheckNotificationVerbosity verbosity,
+                string recipientEmail, string subject)
+            : base(enabled, failureOnly, verbosity)
+        {
+            RecipientEmail = recipientEmail;
+            Subject = subject;
+            Verbosity = verbosity;
+        }
+
+        public string RecipientEmail { get; set; }
+
+        public string Subject { get; set; }
+
+        public async Task SendAsync(HealthCheckResults results)
+        {
+            if (ShouldSend(results) == false)
+            {
+                return;
+            }
+
+            if (string.IsNullOrEmpty(RecipientEmail))
+            {
+                return;
+            }
+
+            using (var client = new SmtpClient())
+            using (var mailMessage = new MailMessage())
```

Comment:
there's no From email addres which should be UmbracoConfig.For.UmbracoSettings().Content.NotificationEmailAddress, i'll fix up